pkg/storage/v2/bigquery/metrics.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2022 The Bucketeer Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bigquery
+
+import (
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/bucketeer-io/bucketeer/pkg/metrics"
+)
+
+const (
+	operationQuery = "Query"
+
+	codeOK      = "OK"
+	codeUnknown = "Unknown"
```

Comment:
Can you also add the following codes?

- backendError
- internalError
- duplicate
- notFound
- jobBackendError
- jobInternalError

Error code reference:
https://cloud.google.com/bigquery/docs/error-messages

Implementation reference:
https://github.com/bucketeer-io/bucketeer/blob/main/pkg/storage/v2/mysql/metrics.go#L60-L79